tox.ini

```diff
@@ -22,6 +22,26 @@ passenv =
     SSL_CERT_FILE  # https proxies
 # recreate = True
 
+[testenv:build-dists]
+basepython = python3
+isolated_build = true
+# `usedevelop = true` overrides `skip_install` instruction, it's unwanted
+usedevelop = false
+skip_install = true
+deps =
+    pep517 >= 0.5.0
+setenv =
+    PYPI_UPLOAD = true
```

Comment:
We shouldn't set this, only bypass the var from outside. This is necessary for proper validation. It must fail when wrongfully trying to upload to PyPI.